code-challenges.test.js

```diff
@@ -10,34 +31,61 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
-
-// a) Create a test with expect statements for each of the variables provided.
-
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
-
-
-// b) Create the function that makes the test pass.
-
+// --------------------Replaced the Instructor Example With the Template
 
+// // a describe method that lists the name of the function OR naming of the particular test.
+// describe("hello", () => {
+//     // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+//     it("returns a string that says hi", () => {
+//       //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+//       expect(hello()).toEqual("hi");
+//     });
+//   });
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
+
 // a) Create a test with expect statements for each of the variables provided.
 
+
 const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
 const object2 =  { number: 0 }
 // Expected output: "0 is divisible by three"
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+describe("divThree", () => {
+    it("determines if the number is divisible by 3 or not and tells you", () => {
+      expect(divThree(object1.number)).toEqual(`15 is divisible by three`);
```

Comment:
glad you are thinking about how to abstract the value. This logic would be best suited for your function. The idea of the expect statement is to show a function call with an argument and the expected output. Logic is handled in the function. 
So the expect statements should look like this
`expect(demarcateThrice(object1)).toEqual("15 is divisible by three")`